lib/routes/watasuke/blog.js

```diff
@@ -0,0 +1,44 @@
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+import cheerio from 'cheerio';
+
+const baseUrl = 'https://watasuke.net';
+
+export default async (ctx) => {
+    const blogPageHtml = await got(`${baseUrl}/blog/`).text();
+    const $ = cheerio.load(blogPageHtml);
+    const ogTitle = $('meta[property="og:title"]').attr('content');
+
+    const apiUrl = `${baseUrl}/page-data/blog/page-data.json`;
+    const response = await got(apiUrl);
+    const articles = response.data.result.data.allArticles.nodes;
+
+    const itemsPromises = articles.map(async (article) => {
+        const articleUrl = `${baseUrl}/blog/article/${article.slug}`;
+        const articleHtml = await got(articleUrl).text();
```

Comment:
The full article body in Markdown is already provided from
https://github.com/DIYgod/RSSHub/blob/9ea5161178385d289cf29458fe8f645fd8352460/lib/routes/watasuke/blog.js#L12
No need to visit the site again for the same content.